CMakeLists.txt

```diff
@@ -520,7 +520,11 @@ list(APPEND HWY_TEST_FILES
 endif()  # HWY_ENABLE_CONTRIB
 
 if(HWY_SYSTEM_GTEST)
-  set(HWY_GTEST_LIBS GTest::GTest GTest::Main)
+  if (CMAKE_VERSION VERSION_LESS 3.20)
```

Comment:
NWM, it is just libjxl cmake files have to be updated as well.

Reply:
@eustas Here is what the doc states:

> Deprecated since version 3.20: For backwards compatibility, this module defines additionally the following deprecated [IMPORTED](https://cmake.org/cmake/help/latest/prop_tgt/IMPORTED.html#prop_tgt:IMPORTED) targets (available since 3.5):
> 
> GTest::GTest
> The Google Test gtest library, if found; adds Thread::Thread automatically
> 
> GTest::Main
> The Google Test gtest_main library, if found
> 

ref:
* https://cmake.org/cmake/help/latest/module/FindGTest.html